src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentTransactionInfoPlugin.java

```diff
@@ -436,4 +412,13 @@ private static void computeLengthArray(final String className, final int[] dotAr
         final Iterable<PluginProperty> propertiesWithDefaults = PluginProperties.merge(defaultProperties, originalProperties);
         return ImmutableList.<PluginProperty>copyOf(propertiesWithDefaults);
     }
+
+    @VisibleForTesting
+    static String formatErrorMessage(final String gatewayError) {
+        if(Strings.isNullOrEmpty(gatewayError)) {
+            return gatewayError;
+        }
+        //Remove any extra spaces between word and convert all to lower-case
+        return gatewayError.replaceAll("\\s+", " ").trim().toLowerCase();
```

Comment:
Looking at https://github.com/killbill/killbill-payment-retries-plugin/pull/1, we don't need this anymore, do we?

Reply:
You mean the format part? Yes, that is not that necessary.

Reply:
Removed in b2ac7f3